tendrl/gluster_integration/sds_sync/__init__.py

```diff
@@ -684,3 +698,80 @@ def sync_volumes(volumes, index, vol_options, sync_ttl):
             b_index += 1
         except KeyError:
             break
+
+
+def brick_status_alert(hostname):
+    # fetching brick details of disconnected node
+    bricks = NS._int.client.read(
+        "clusters/%s/Bricks/all/%s" % (
+            NS.tendrl_context.integration_id,
+            hostname
+        )
+    )
+    for brick_info in bricks.leaves:
+        try:
+            brick_path = brick_info.key
+            lock = None
+            lock = etcd.Lock(
+                NS._int.client,
+                brick_path[1:]
+            )
+            lock.acquire(
+                blocking=True,
+                lock_ttl=60
+            )
+            if lock.is_acquired:
+                brick = NS.gluster.objects.Brick(
+                    fqdn=hostname,
+                    brick_dir=brick_path.split("/")[-1]
+                ).load()
+                if brick.status == BRICK_STATUS[0]:
```

Comment:
If this piece of code gets executed only for disconnected hosts, then I feel you can safely say that brick is stopped.

Reply:
replace array by constant?

Reply:
replace this statement by brick_stop constant right?

Reply:
yes